cmd/repositories_test.go

```diff
@@ -0,0 +1,31 @@
+/*
+* Copyright 2021 LimeChain Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+ */
+
+package main
+
+import (
+	"github.com/limechain/hedera-eth-bridge-validator/test/mocks"
+	"github.com/stretchr/testify/assert"
+	"gorm.io/gorm"
+	"testing"
+)
+
+func TestPrepareRepositories(t *testing.T) {
+	mocks.Setup()
+	mocks.MDatabase.On("GetConnection").Return(&gorm.DB{})
+	res := PrepareRepositories(mocks.MDatabase)
+	assert.NotEmpty(t, res)
```

Comment:
Maybe we should be asserting that the proper repositories are prepared. Even if they are just mocked(or empty).